### ISSUE TITLE
Large amount of wasted memory on font files in Firefox

### ISSUE THREAD

**Issue by WGH-** [FelisCatus/SwitchyOmega#1869]
Reopening new issue as requested in https://github.com/FelisCatus/SwitchyOmega/issues/1330#issuecomment-403368592.

I also noticed that this appears only after some browser use (several days or so).

I'm running Proxy SwitchyOmega 2.5.20 on Firefox 60.8.0 on Linux.

According to about:memory page, the addon wastes quite a lot of memory on base64-encoded font files.
```
│  ├──149.18 MB (16.15%) -- switchyomega@feliscatus.addons.mozilla.org
│  │  ├──149.17 MB (16.15%) -- window-objects/top(moz-extension://074dcd7d-9b54-45eb-a95b-010f0825867f/background.html, id=56)
│  │  │  ├──129.19 MB (13.99%) -- js-zone(0x7f86bf23b000)
│  │  │  │  ├──111.73 MB (12.10%) -- strings
│  │  │  │  │  ├───63.43 MB (06.87%) -- (301 tiny)
│  │  │  │  │  │   ├───8.75 MB (00.95%) ++ string(length=62526, copies=140, "data:application/font-woff;base64,d09GRgABAAAAALcTAA4AAAABY9AAAAAAAAC1qAAAAWsAAALYAAAAAAAAAABPUy8yAAABnAAAAFgAAABgjga5umNtYXAAAAH0AAAG6AAAEgCbcKQqY3Z0IAAAs/gAAAAoAAAAKAjhCLJmcGdtAAC0IAAAANAAAAEs84Klmmdhc3AAALPoAAAAEAAAABAAGgAjZ2x5ZgAALmQAAIWEAAEDFE+Ll4hoZWFkAAABZAAAADYAAAA2CfaG5WhoZWEAAAjcAAAAIQAAACQH6Ae4aG10eAAACQAAAAaSAAAR9o9pil9sb2NhAAAlxAAACJ8AAAj+ol1dpm1heHAAAAFEAAAAIAAAACAE+ATJbmFtZQAAD5QAAAG0AAAEQXAVNDJwb3N0AAARSAAAFHoAACt3rYf6aXByZXAAALTwAAAAtwAAANzHz3zdAAEAAAR+AGAABwBgAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxLy8SDRfDzz1AAMD6AAAAADTwC3PAAAAANQAFJr+i/6dBJIENwAAAAcAAgAAAAAAAHjaY2Bm0mScwMDKwMLUxRTBwMDgDaEZ4xhEGBUZGJgYWNmYQRSLAgNTO1CekQEKcnOKixkcGBh+szDP+K/HaMA8g+GSAgPjfJAc42Om2UBKgYEZAAzUDP142u1WB3NVxxk9Z79PgApVEoh2ue9hid4NohtME92UYGeSIbbHYEyxAePE2LQ4EAc7dLDpFr0pMcVgEBJF9N5Ff0h6mGJjOo/OyyIrECbJ5AdEe+fb/XZ3du6dc/c75wAwAMRGlO0JhIZB8wUA5rGTvMhjd/KjOpqgFd5CT/TFMA7nOE7kLM7hcl7kTwyakqaWiTe/NgPNcpNidphdZo85KyoRUl6GyZ/lC/mrjJfJMkeWyHJJkf1KjdCCGq3VtbZ21E76W/2dTtIZukovO+KEO2Wcss7rzltOH2eUM9u574a50a7jet04t7pby63vNnabu" (truncated))
│  │  │  │  │  │   ├───8.75 MB (00.95%) ++ string(length=63162, copies=140, "data:application/font-woff;base64,d09GRgABAAAAALjxAA4AAAABY7AAAAAAAAC3iAAAAWkAAALUAAAAAAAAAABPUy8yAAABnAAAAFgAAABgjhO5w2NtYXAAAAH0AAAG6AAAEgCbcKQqY3Z0IAAAtdgAAAAoAAAAKAkBCMBmcGdtAAC2AAAAANAAAAEs84Klmmdhc3AAALXIAAAAEAAAABAAGgAjZ2x5ZgAALmAAAIdnAAEDGCHBWq5oZWFkAAABZAAAADYAAAA2Cc5QaWhoZWEAAAjcAAAAIQAAACQIAwfMaG10eAAACQAAAAaXAAAR9rQtdihsb2NhAAAlwAAACJ0AAAj+mMNT+G1heHAAAAFEAAAAIAAAACAE+ATJbmFtZQAAD5gAAAGrAAAEHYmAkptwb3N0AAARRAAAFHoAACt3rYf6aXByZXAAALbQAAAAtwAAANzHz3zdAAEAAAR+AGAABwBgAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxK7mro5fDzz1AAMD6AAAAADTwC3PAAAAANPJ3iv+f/6QBKwENwAAAAcAAgAAAAAAAHjaY2BmMmKcwMDKwMLUxRTBwMDgDaEZ4xhEGBUZGJgYWNmYQRSLAgNTO1CekQEKcnOKixkcGBh+szDP+K/HaMs8g+GSAgPjfJAc42Om2UBKgYEZABFEDRN42u1WB3NVxxk9Z79PgApVEoh2ue9hid4NohtME92UYGeSIbbHYEyxAePE2LQ4EAc7dLDpFr0pMcVgEBJF9N5Ff0h6mGJjOo/OyyIrECbJ5AdEe+fb/XZ3du6dc/c75wAwAMRGlO0JhIZB8wUA5rGTvMhjd/KjOpqgFd5CT/TFMA7nOE7kLM7hcl7kTwyakqaWiTe/NgPNcpNidphdZo85KyoRUl6GyZ/lC/mrjJfJMkeWyHJJkf1KjdCCGq3VtbZ21E76W/2dTtIZukovO+KEO2Wcss7rzltOH2eUM9u574a50a7jet04t7pby63vNnabu" (truncated))
│  │  │  │  │  │   ├───8.75 MB (00.95%) ++ string(length=63198, copies=140, "data:application/font-woff;base64,d09GRgABAAAAALkLAA4AAAABZUwAAAAAAAC3oAAAAWsAAALXAAAAAAAAAABPUy8yAAABnAAAAFgAAABgjfO5sGNtYXAAAAH0AAAG6AAAEgCbcKQqY3Z0IAAAtfAAAAAoAAAAKAiyCJlmcGdtAAC2GAAAANAAAAEs84Klmmdhc3AAALXgAAAAEAAAABAAGgAjZ2x5ZgAALlwAAIeDAAEEtIo/tthoZWFkAAABZAAAADYAAAA2Ca9QlmhoZWEAAAjcAAAAIQAAACQHxAeeaG10eAAACQAAAAaWAAAR9mCMpmJsb2NhAAAlvAAACJ4AAAj+fhw42m1heHAAAAFEAAAAIAAAACAE+ATJbmFtZQAAD5gAAAGnAAAEIPrebOFwb3N0AAARQAAAFHoAACt3rYf6aXByZXAAALboAAAAtwAAANzHz3zdAAEAAAR+AGAABwBgAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxNlYD91fDzz1AAMD6AAAAADTwC3PAAAAANPJ3jj+nP6wBHAENwAAAAcAAgAAAAAAAHjaY2BmkmecwMDKwMLUxRTBwMDgDaEZ4xhEGBUZGJgYWNmYQRSLAgNTO1CekQEKcnOKixkcGBh+szDP+K/HKMs8g+GSAgPjfJAc42Om2UBKgYEZAAd9DOB42u1WB3NVxxk9Z79PgApVEoh2ue9hid4NohtME92UYGeSIbbHYEyxAePE2LQ4EAc7dLDpFr0pMcVgEBJF9N5Ff0h6mGJjOo/OyyIrECbJ5AdEe+fb/XZ3du6dc/c75wAwAMRGlO0JhIZB8wUA5rGTvMhjd/KjOpqgFd5CT/TFMA7nOE7kLM7hcl7kTwyakqaWiTe/NgPNcpNidphdZo85KyoRUl6GyZ/lC/mrjJfJMkeWyHJJkf1KjdCCGq3VtbZ21E76W/2dTtIZukovO+KEO2Wcss7rzltOH2eUM9u574a50a7jet04t7pby63vNnabu" (truncated))
│  │  │  │  │  │   ├───8.75 MB (00.95%) ++ string(length=63230, copies=140, "data:application/font-woff;base64,d09GRgABAAAAALklAA4AAAABZegAAAAAAAC3vAAAAWkAAALVAAAAAAAAAABPUy8yAAABnAAAAFgAAABgjeW5qGNtYXAAAAH0AAAG6AAAEgCbcKQqY3Z0IAAAtgwAAAAoAAAAKAiQCIVmcGdtAAC2NAAAANAAAAEs84Klmmdhc3AAALX8AAAAEAAAABAAGgAjZ2x5ZgAALnAAAIeKAAEFSNlYQZJoZWFkAAABZAAAADYAAAA2CdGEtGhoZWEAAAjcAAAAIQAAACQHogeNaG10eAAACQAAAAajAAAR9j47uudsb2NhAAAl0AAACJ8AAAj+0pqNQG1heHAAAAFEAAAAIAAAACAE+ATJbmFtZQAAD6QAAAGvAAAEJhmpQCRwb3N0AAARVAAAFHoAACt3rYf6aXByZXAAALcEAAAAtwAAANzHz3zdAAEAAAR+AGAABwBgAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxJmWFwlfDzz1AAMD6AAAAADTwC3PAAAAANQAEkj+of6+BFcENwAAAAcAAgAAAAAAAHjaY2BmEmecwMDKwMLUxRTBwMDgDaEZ4xhEGBUZGJgYWNmYQRSLAgNTO1CekQEKcnOKixkcGBh+szDP+K/HyM88g+GSAgPjfJAc42Om2UBKgYEZAANTDMp42u1WB3NVxxk9Z79PgApVEoh2ue9hid4NohtME92UYGeSIbbHYEyxAePE2LQ4EAc7dLDpFr0pMcVgEBJF9N5Ff0h6mGJjOo/OyyIrECbJ5AdEe+fb/XZ3du6dc/c75wAwAMRGlO0JhIZB8wUA5rGTvMhjd/KjOpqgFd5CT/TFMA7nOE7kLM7hcl7kTwyakqaWiTe/NgPNcpNidphdZo85KyoRUl6GyZ/lC/mrjJfJMkeWyHJJkf1KjdCCGq3VtbZ21E76W/2dTtIZukovO+KEO2Wcss7rzltOH2eUM9u574a50a7jet04t7pby63vNnabu" (truncated))
│  │  │  │  │  │   ├───6.58 MB (00.71%) ++ string(length=67666, copies=99, "data:application/font-woff;base64,d09GRgABAAAAAMYjAA4AAAABfswAAAAAAADEuAAAAWsAAALaAAAAAAAAAABPUy8yAAABnAAAAFsAAABgjfS57WNtYXAAAAH4AAAG4wAAEfCbVKQHY3Z0IAAAwwgAAAAoAAAAKAiyCJlmcGdtAADDMAAAANAAAAEs84Klmmdhc3AAAML4AAAAEAAAABAAGgAjZ2x5ZgAAL0wAAJOqAAEeKEjE5vhoZWFkAAABZAAAADYAAAA2CfVRCGhoZWEAAAjcAAAAIwAAACQIqgsmaG10eAAACQAAAAdkAAAR9mFhb6Vsb2NhAAAmnAAACK0AAAj+rM5gxm1heHAAAAFEAAAAIAAAACAE+AThbmFtZQAAEGQAAAG3AAAEORfZpb5wb3N0AAASHAAAFH4AACt3rXt6aXByZXAAAMQAAAAAtwAAANzHz3zdAAEAAAR+AGwABwBsAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxME/07xfDzz1AAMD6AAAAADTwC45AAAAANPJ3kD+2v6wBHgENwAAAAcAAgAAAAAAAHjaY2BmkmecwMDKwMLUxRTx/z2DN4hm8GOMYxBhVGRgYGJgZWMGUSwKDEztDAwMjAxQkJtTXMzgwMDwm4V5xn89RlnmGQyXFBgY54PkGB8zzQZSCgzMALsIDxwAeNrtlndwFdcVxr/vniNAhSoJRFv2PSzRi8AgusE00U0JdiYZYnsMxhQbME6MTYsDcbBDB5tu0ZsSUwwGIVFE7110Ielhio3pPDovV7ICYZJM/s/o7pxbZ2d3frvnOx8AA0BsRNieQLCBFrgGUO0iP9Se1ERTtMZb6IV+GM4RHM9JnM25XMFL/IkBU9rUNnHm12aQWWGSzU6z2+w150QlTCrKcPmzfCF/lQkyRebKUlkhyXJAqWFaWCO1ptbRTtpZf6u/08k6U1frFUecUKecU9553XnL6euMduY4D9wQN9J1XK8b49Z0a7sN3CZuC" (truncated))
│  │  │  │  │  │   ├───2.42 MB (00.26%) ++ string(length=24028, copies=103, "data:application/font-woff;charset=utf-8;base64,d09GRk9UVE8AAEZAAAoAAAAAfBAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAABDRkYgAAAA9AAAQsUAAHZfa1y5A0ZGVE0AAEO8AAAAGQAAABx4Dt9ZT1MvMgAAQ9gAAABKAAAAYEC7Yj5jbWFwAABEJAAAAIUAAAG6pEWcoGhlYWQAAESsAAAALwAAADYHEbeJaGhlYQAARNwAAAAdAAAAJAOvAd5obXR4AABE/AAAADgAAABeCDEE521heHAAAEU0AAAABgAAAAYALVAAbmFtZQAARTwAAADrAAAB5koHYmpwb3N0AABGKAAAABYAAAAg/8MAGnicrZ13mJXFFfDn3XbvVnb37i596SC9d8uLDQyKFXtD7F0RYzSGay9LDCpqjB1REaPR2CXCFbGB2ACR3otLWdje5zu/M++9QGK+fH98Dzyz804vZ86cOtczKSnG87zsKTdMvmrStX2vveGKG6YYL8l45qTaFqZ2rFc7Lqn2hOTaVinTs7zibW36ZyUXd3vVZqUUZ5jWp+fbkpJEJCv04mW102p7pbZLat2inTG57ZJuzmtn+rSb2jLf9KfJsGlhikx709X0NoPMSOObsWaCmWjON5PN1eYmc5uZZu43fzYzzTNmlnndvG0+NPPNF+Zbs8ysNpvMTlNmqk2Tl+Jlevlea6+j18Pr5w2Vf2O8k6Zef9XxAwcM5M+gAQPcn+BrkPsz2P0Z6v4Mc3+Guz8j9c9AV2+gqzfQ1Rvo6g0c4v646gNd9YGu+sAR7o9rZZBrZZCrPshVH+SqD3LVB2n1Ia6/Ia6HIa7pIcO0laHua6jraJib0TA3o2FuRsPcjIYNGOL+DHV/ggrD3Z8R7o+2Ocz1N2zgMfpn0OCDd/0QCDDGe8B70HvIe9gr8aZ7f/Ye8f7izfAe9R7zHvdmek94T3pPeX/1nvb+5j3jPes95z3vveC96L3kzfJe9mZ7r3iveq95c7zXvbn" (truncated))
│  │  │  │  │  │   ├───1.01 MB (00.11%) ++ string(length=1056134, copies=1, "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA2AAAAG8CAYAAAC48cbZAAAABHNCSVQICAgIfAhkiAAAIABJREFUeF7MvXnQpddZ4Pfc/X7323pf1Fq6JVmbZXlfJNnYxjYMW7xCKAZDUZNZKmFqUkwqqUwFipqaqhQ1k/wxmaJSk7AkATIMdioMDIYxNhgb7zLGNpa1S61u9b58+91vfr/n3Lf1qSU7hoAz5+vb977ve9bnPPt5znlrH/2Vw7Mj/QuxenoxGme34/zDk2icjFhej6gNImatiPZyLVoLzRiPR3mvzb1ZoxaDGnn421mfxnQY0WpyPY2I7Yjm2Gfc60RM6hGjnYhunTutWbR7tdjcnvE0orFI+ZjFbCuiTtkp5awiyzabMYpJXJrN4oZX0+7bIxaON6NzJ5Xu2YnpaBpWOZk2YjKZRaNBvyxo+VqNa25Q22xK/fxNqbhOX+o1/vMJNybey+tZ1KisTrnReJLPLZ7PqHM8ZozU02DcZMn+Rf72F5XUGI+/HZYfx8JnRt+jTjm6nPdH5Onvj/h6M/q/ez5OfmQaLWFnfaR59/N7XnMWq1Lp+a4b85/zJvNqd/4X5/z279iWEPSbaYmxP26KOPhgPXp895vTGAKq0Qh8Wakn/MbMSQ24NFt82gKQQgln4D+ZZJ5GvRFccs19Po1GPZpNWyrzNhwy68xnwnkOkMl4Vua4yRwBmToV+KjBNeArdVD3ZDSJ0dC54iZArYmT9NGu572ss1Sa0zUHlvNqP8SJMW2ZvGdyCPZ5TN3igdetlnlLn9rtetYzGvIf/8ShUkcNfAZGwMfUsS8Ms0WfzTiSRubj8/mAfBPuiXetNs/sNM+btCWej6hLfM0yfBxH4hf/anTKfgm37Az/OsDfPEMmyf743LKDPnMJTKRj6bMu3tMnYTnq0wbjsOnsH31pNusx3mGehKMwhb5nEP/E70tQFm0tHGhEvcPDbRgE+A6xZyelu/FgFi0LUtegP6SOOvBrxmSDm" (truncated))
│  │  │  │  │  │   ├───0.78 MB (00.08%) ── string(length=11, copies=34041, "127.0.10.80")/gc-heap/latin1
│  │  │  │  │  │   ├───0.78 MB (00.08%) ── string(length=5, copies=34036, "socks")/gc-heap/latin1
│  │  │  │  │  │   ├───0.73 MB (00.08%) ── string(length=3, copies=32107, "GET")/gc-heap/latin1
│  │  │  │  │  │   ├───0.58 MB (00.06%) ── string(length=7, copies=25169, "0.0.0.0")/gc-heap/latin1
(...)
│  │  │  │  │  ├───14.31 MB (01.55%) ++ string(<non-notable strings>)
│  │  │  │  │  ├───12.07 MB (01.31%) ++ string(length=119470, copies=103, "data:font/woff;base64,d09GRgABAAAAAV3wABIAAAACw4QAAgR+AAAAAAAAAAAAAAAAAAAAAAAAAABGRlRNAAABlAAAABwAAAAcYgkChUdERUYAAAGwAAAAZgAAAHopcybcR1BPUwAAAhgAADFTAACDLI1iZg5HU1VCAAAzbAAACqMAABSexz3ey09TLzIAAD4QAAAAVgAAAGCh07GaY21hcAAAPmgAAANIAAAErjOWwQZjdnQgAABBsAAAAEIAAABCBE4k2GZwZ20AAEH0AAABOwAAAbxT71erZ2FzcAAAQzAAAAAMAAAADAAIABNnbHlmAABDPAAA7ZYAAc58Ibs0ImhlYWQAATDUAAAANAAAADYGmlMfaGhlYQABMQgAAAAhAAAAJAsmCq5obXR4AAExLAAACmwAABOIVOwKJmxvY2EAATuYAAAJnwAACcZJJ9CWbWF4cAABRTgAAAAgAAAAIAcPAhhuYW1lAAFFWAAAAlQAAASGQuBcnnBvc3QAAUesAAAVtwAAMWPsiHvtcHJlcAABXWQAAACKAAAAot02fhoAAAABAAAAAMw9os8AAAAAxPARLgAAAADQ206HeJwdjUEKwgAQAyebPYuPqfYzrejF3j1YsQVFwfpNf+JqQmDIIUHAqnLip2CNuFamspnVIfXqsXbaFx90LB6iQbGJLY7WZ+TRI/bFE+HZt2rufhQ/84VyyYXId35qOf8/fAERxg6SAAB4nO2dCXxURbb/T9UN2UNCCAHXcYMAbig6KoKA8wSVWf7gyN8FRMflufCUUQfHcVdUBpXnMoNxFMImMrLMwICACrTIjiExQAdiCHSn090kne5OOtE3T0z9v/d2JySC/pn5v/+893kvqc/vVt17695bdX7nnDp1u6sjSkTSZZxMlqR/GvHjn8uJtz/64ETp+88P3nmf/HDibQ/fL/8kSdQRY0STqb9hT98+6aFJ0vW+Ox+8X/KcI+JsOSPJ0tXZV5Lv1E5Sey+794pf3/0CR9Kli5wlV1Oqk" (truncated))
│  │  │  │  │  ├───12.07 MB (01.31%) ++ string(length=119666, copies=103, "data:font/woff;base64,d09GRgABAAAAAV6EABIAAAACvqgAAgRNAAAAAAAAAAAAAAAAAAAAAAAAAABGRlRNAAABlAAAABwAAAAcYgkCmEdERUYAAAGwAAAAZgAAAHopdSbeR1BPUwAAAhgAAC6BAAB0rl9Hb9hHU1VCAAAwnAAACqUAABSexqfgUE9TLzIAADtEAAAAVwAAAGCgp7GiY21hcAAAO5wAAANIAAAErjOWwQZjdnQgAAA+5AAAAEwAAABMJEEG5WZwZ20AAD8wAAABOwAAAbxn9FyrZ2FzcAAAQGwAAAAMAAAADAAIABNnbHlmAABAeAAA8KkAAdfo0ESDPmhlYWQAATEkAAAAMwAAADYGLVnvaGhlYQABMVgAAAAhAAAAJAq6CoJobXR4AAExfAAACnEAABOI6ISIlGxvY2EAATvwAAAJpgAACcaWnhz8bWF4cAABRZgAAAAgAAAAIAcPAqNuYW1lAAFFuAAAAlUAAAR6doFSlXBvc3QAAUgQAAAVtAAAMWKKISvdcHJlcAABXcQAAAC9AAAA23Sgj+wAAAABAAAAAMw9os8AAAAAxPARLgAAAADQ206aeJwdjTEKwgAUQ19+/iweptrLtKKL3R2s2IKig/WY3sSvCYFHhgQBq8qJn4I14lqZymZWh9Srx9ppX3zQsXiIBsUmtjhan5FHj9gXT4Rn36q5+1H8zBfKJRci3/mp5fz/8AUSDg6WAAB4nO2dCXwUVb7v//9TCQmhQ8KSEBlEVMIiuDI6qKjgXAXBGS+ovBFlxFHfHZfBGZ1xX6KiICKDMipu7CKjwGQUBFQQkbAEAglLCLEJEJruhnTSnU3HK+a8b1UWwqJX57137/18bnI+vzqnq07VOf/l/JfqqrSoiCTJTXK/xP3LlVdfJ51vfejeu6XXv917+11ywd23/HGs/IvE0UesFUOlP+KTufX39/1e2t51+71jpaO3R7wtR6SVtPU+q6R7veO06Ly4/gNHJbInSeKluwyhVSKJkik9p" (truncated))
│  │  │  │  │  └────9.85 MB (01.07%) ++ string(length=119774, copies=84, "data:font/woff;base64,d09GRgABAAAAAV7UABIAAAACvhAAAgSAAAAAAAAAAAAAAAAAAAAAAAAAAABGRlRNAAABlAAAABwAAAAcYgkCkkdERUYAAAGwAAABNAAAAcRmFmBcR1BPUwAAAuQAAC6KAABeTnsohtlHU1VCAAAxcAAADPsAADDo6mgGW09TLzIAAD5sAAAAVwAAAGChC7GwY21hcAAAPsQAAANIAAAErjOWwQZjdnQgAABCDAAAAFYAAABWBJcrSmZwZ20AAEJkAAABOwAAAbx7+WGrZ2FzcAAAQ6AAAAAMAAAADAAIABNnbHlmAABDrAAA7cYAAc+Qp89n62hlYWQAATF0AAAAMwAAADYGYlmraGhlYQABMagAAAAhAAAAJArvCptobXR4AAExzAAACl4AABOIJPNFAGxvY2EAATwsAAAJoAAACcZg7ufmbWF4cAABRcwAAAAgAAAAIAcPAp9uYW1lAAFF7AAAAoEAAAUQVmT86nBvc3QAAUhwAAAVtwAAMWPsiHvtcHJlcAABXigAAACrAAAAzBux+DYAAAABAAAAAMw9os8AAAAAxPARLgAAAADQ206UeJwd0UlL1WEUx/HfOc8/8bZoaeCQWzV9BZZbF7qz16KBgnKvGThlGA44mwPOiqJJ16GNcwn5CoRScNg6o371fHng8+wOHJmkV5Il9TSuYpnivIRq+X0iUx25PqseN6gRN6kZf1EL/qpW/E1tuJ1MHeTqVBfuVg/uVR/uJ9eABvF3DeFhMo2Qa1TjeEKTeErTeEazeE7zeEGLeIlMP8i1rJ84qRW8qjW8Tq5f2sLb2sG7+of/k+tIJ/hUZ/hcV/haN/iWTHd6kJvshcxSLIZfWhp+bek4wzJxlr3B2ZaH31o+LrBC/M7e4yIrwaVWhj/4hsw3fU/uv/0P3ve/+MCP8LGzg5/7Bb70e/wQUmUhFnLkITdU4I+hEleFalwT4jgRlcuiimhMHo1Hh+wcPV9Mj+3tVOV4nO2ceXxU1f33v" (truncated))
│  │  │  │  ├───11.34 MB (01.23%) ── unused-gc-things
│  │  │  │  └────6.11 MB (00.66%) ++ (14 tiny)
│  │  │  └───19.98 MB (02.16%) ++ active/window(moz-extension://074dcd7d-9b54-45eb-a95b-010f0825867f/background.html)
│  │  └────0.02 MB (00.00%) ++ js-non-window/zones/zone(0x7f86f247d000)/compartment(moz-extension://074dcd7d-9b54-45eb-a95b-010f0825867f/, moz-extension://074dcd7d-9b54-45eb-a95b-010f0825867f/background.html)
```

**Comment by WGH-** [comment]
Also, not sure if it's related to the same problem, but there're quire a few short string with large amount of copies, which result in relatively large total size:
```
0.79 MB (00.07%) ── string(length=11, copies=34424, "127.0.10.80")/gc-heap/latin1
0.79 MB (00.07%) ── string(length=5, copies=34419, "socks")/gc-heap/latin1
0.74 MB (00.07%) ── string(length=3, copies=32413, "GET")/gc-heap/latin1
0.58 MB (00.05%) ── string(length=7, copies=25388, "0.0.0.0")/gc-heap/latin
0.33 MB (00.03%) ── string(length=1, copies=14370, ".")/gc-heap/latin1
0.29 MB (00.03%) ── string(length=1, copies=12620, "r")/gc-heap/latin1
```

**Comment by WGH-** [comment]
The leak seems to be slowly progressing over time
```
│    ├──204.99 MB (19.36%) -- switchyomega@feliscatus.addons.mozilla.org
│    │  ├──204.97 MB (19.36%) -- window-objects/top(moz-extension://074dcd7d-9b54-45eb-a95b-010f0825867f/background.html, id=56)
│    │  │  ├──179.60 MB (16.97%) -- js-zone(0x7f86bf23b000)
│    │  │  │  ├──173.05 MB (16.35%) -- strings
│    │  │  │  │  ├───52.57 MB (04.97%) -- (398 tiny)
│    │  │  │  │  │   ├──10.56 MB (01.00%) ++ string(length=67666, copies=159, "data:application/font-woff;base64,d09GRgABAAAAAMYjAA4AAAABfswAAAAAAADEuAAAAWsAAALaAAAAAAAAAABPUy8yAAABnAAAAFsAAABgjfS57WNtYXAAAAH4AAAG4wAAEfCbVKQHY3Z0IAAAwwgAAAAoAAAAKAiyCJlmcGdtAADDMAAAANAAAAEs84Klmmdhc3AAAML4AAAAEAAAABAAGgAjZ2x5ZgAAL0wAAJOqAAEeKEjE5vhoZWFkAAABZAAAADYAAAA2CfVRCGhoZWEAAAjcAAAAIwAAACQIqgsmaG10eAAACQAAAAdkAAAR9mFhb6Vsb2NhAAAmnAAACK0AAAj+rM5gxm1heHAAAAFEAAAAIAAAACAE+AThbmFtZQAAEGQAAAG3AAAEORfZpb5wb3N0AAASHAAAFH4AACt3rXt6aXByZXAAAMQAAAAAtwAAANzHz3zdAAEAAAR+AGwABwBsAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxME/07xfDzz1AAMD6AAAAADTwC45AAAAANPJ3kD+2v6wBHgENwAAAAcAAgAAAAAAAHjaY2BmkmecwMDKwMLUxRTx/z2DN4hm8GOMYxBhVGRgYGJgZWMGUSwKDEztDAwMjAxQkJtTXMzgwMDwm4V5xn89RlnmGQyXFBgY54PkGB8zzQZSCgzMALsIDxwAeNrtlndwFdcVxr/vniNAhSoJRFv2PSzRi8AgusE00U0JdiYZYnsMxhQbME6MTYsDcbBDB5tu0ZsSUwwGIVFE7110Ielhio3pPDovV7ICYZJM/s/o7pxbZ2d3frvnOx8AA0BsRNieQLCBFrgGUO0iP9Se1ERTtMZb6IV+GM4RHM9JnM25XMFL/IkBU9rUNnHm12aQWWGSzU6z2+w150QlTCrKcPmzfCF/lQkyRebKUlkhyXJAqWFaWCO1ptbRTtpZf6u/08k6U1frFUecUKecU9553XnL6euMduY4D9wQN9J1XK8b49Z0a7sN3CZuC" (truncated))
│    │  │  │  │  │   ├───3.40 MB (00.32%) ++ string(length=24028, copies=145, "data:application/font-woff;charset=utf-8;base64,d09GRk9UVE8AAEZAAAoAAAAAfBAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAABDRkYgAAAA9AAAQsUAAHZfa1y5A0ZGVE0AAEO8AAAAGQAAABx4Dt9ZT1MvMgAAQ9gAAABKAAAAYEC7Yj5jbWFwAABEJAAAAIUAAAG6pEWcoGhlYWQAAESsAAAALwAAADYHEbeJaGhlYQAARNwAAAAdAAAAJAOvAd5obXR4AABE/AAAADgAAABeCDEE521heHAAAEU0AAAABgAAAAYALVAAbmFtZQAARTwAAADrAAAB5koHYmpwb3N0AABGKAAAABYAAAAg/8MAGnicrZ13mJXFFfDn3XbvVnb37i596SC9d8uLDQyKFXtD7F0RYzSGay9LDCpqjB1REaPR2CXCFbGB2ACR3otLWdje5zu/M++9QGK+fH98Dzyz804vZ86cOtczKSnG87zsKTdMvmrStX2vveGKG6YYL8l45qTaFqZ2rFc7Lqn2hOTaVinTs7zibW36ZyUXd3vVZqUUZ5jWp+fbkpJEJCv04mW102p7pbZLat2inTG57ZJuzmtn+rSb2jLf9KfJsGlhikx709X0NoPMSOObsWaCmWjON5PN1eYmc5uZZu43fzYzzTNmlnndvG0+NPPNF+Zbs8ysNpvMTlNmqk2Tl+Jlevlea6+j18Pr5w2Vf2O8k6Zef9XxAwcM5M+gAQPcn+BrkPsz2P0Z6v4Mc3+Guz8j9c9AV2+gqzfQ1Rvo6g0c4v646gNd9YGu+sAR7o9rZZBrZZCrPshVH+SqD3LVB2n1Ia6/Ia6HIa7pIcO0laHua6jraJib0TA3o2FuRsPcjIYNGOL+DHV/ggrD3Z8R7o+2Ocz1N2zgMfpn0OCDd/0QCDDGe8B70HvIe9gr8aZ7f/Ye8f7izfAe9R7zHvdmek94T3pPeX/1nvb+5j3jPes95z3vveC96L3kzfJe9mZ7r3iveq95c7zXvbn" (truncated))
│    │  │  │  │  │   ├───1.61 MB (00.15%) ++ string(length=12306, copies=103, "data:font/opentype;base64,d09GRgABAAAAACP4AA0AAAAAPkwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABHUE9TAAAZrAAACWsAABig3xTXdUdTVUIAACMYAAAA3QAAAX41JDWOT1MvMgAAGLAAAABTAAAAYHhRPVtjbWFwAAAZBAAAAGwAAACSQ/FjK2dhc3AAABmkAAAACAAAAAj//wAEZ2x5ZgAAATAAABTHAAAezBEpcAFoZWFkAAAXAAAAADYAAAA2/JdmmWhoZWEAABiQAAAAHwAAACQHFQLwaG10eAAAFzgAAAFVAAAB3NbYFN1sb2NhAAAWFAAAAOwAAADwwILJDm1heHAAABX4AAAAHAAAACAAhgCZbmFtZQAAGXAAAAAfAAAAIAAjCJtwb3N0AAAZkAAAABMAAAAg/7gAMnjalVkFYNtI1p4nNbYDNcsKNA4osRx0Ytlxgw6T3dhumJMyt1lul3nbg94yXWmZmelf6jIzHy0zc5T/jaQ0bvewu7Jlzcz3+M2nCWGIkxD4AxMkGSSLECkg6IWAEJCUS9Irl15QLhZ/3eEOuavctUl1qc1JrfmV+WG8WpJaUyNJUXdVaij17rvv9m30XYv/8OtuGNu4kbCkZOYauJ+5muQQN/EQEgQH7y8Fv68i4Jc4J/D6UhA5h+QEzq7Tc4Lfle9ziVwdsHacaISS1qSRnshAqKo9omsyFLrnMT3dw4urzoOSYk9R2YcFhWLxmfaJxFey2jqaehPbWua1VfHFhfOeZ7Ka6iKjhlhUv8fptt9n8MGinGLrHfoyQhKIe+YfzM3MOtRuPuHQcpGQAPDAelEHIdflq7B5/T4hl7Mn/ObJmbDJK5/uOKWlqamptbGxqQkeiLtnxp3Te5itl0WH+3eOyH/bfMghm3XxPwiQEniAnMxcjrKJTZT0teYBMzzw/vs4UjMzCR7mUZKMKqJX0EO8EewO8HiWTi5ZWtrf1HS8M/rer+/GnBuv+fYagisSZibJj+oKXqwDdFzAC" (truncated))
│    │  │  │  │  │   ├───1.61 MB (00.15%) ++ string(length=13018, copies=103, "data:font/opentype;base64,d09GRgABAAAAACYQAA0AAAAARyAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABHUE9TAAAgaAAABPoAABWkBZz+c0dTVUIAACVkAAAArAAAATj9a/zxT1MvMgAAH2wAAABUAAAAYF5zoiVjbWFwAAAfwAAAAGYAAACEY2eEJGdhc3AAACBgAAAACAAAAAj//wAEZ2x5ZgAAATAAABuSAAArFrYFcRpoZWFkAAAdwAAAADYAAAA2824NdGhoZWEAAB9MAAAAIAAAACQH2geOaG10eAAAHfgAAAFRAAABwPKJEmpsb2NhAAAc3AAAAOIAAADiTLlCNm1heHAAABzEAAAAGAAAACAAfwB1bmFtZQAAICgAAAAkAAAAJACmCM1wb3N0AAAgTAAAABMAAAAg/58AMnjalXoFnBpX/vi8x+IwMNjgAwwwOCzDwC67sMJ61iIb2XibS2oXufbqktTdvb3mpJ7/v+7enl/13CtndXdh+L03QJbdJif5hJ33Bubr/oaAxPbap8Tf4B5CR9gJwpwCQo7PWi0KpYylsza0YANhIZf/NgSFAoDujIP2++12vx9GLU7dyIjOaRFP7g/QdADfJRC8BLgR+OAvCD3hQvC4cA4DxIAMgJ23e8cHomYaAN+dzQW48SW5Ss47LHJS/lLLEkHtq70Ob4E/QTAZggBKvkAreSsrfbiC9BF46SNDF/D9qaun05PnXNP+g7uXpaevWppecuHuzHW3Ld1wGX8FvOfFd6kX0xenq8a30X+0eE+cupggZESxdjX4AL5KcESCyCIs1qyNtrJpkAKs1QuwTKyskAKcwDPAC2gKyypfEKwKq4U254VcmAOp8EwyaTA4o85DZjdtiPeuTaRMaqYjumHFhhUHHBDfIXTDV4e12mvXWtWqj70zQ+MHWjZvhjrtzd/UkxoA2pzLh0bWk2PHOGZnvOKt9KZkh4Ug5ART+0gmIB1pCRvhJMJEO5Jsi3LoACKBz2ISQoAF+/nq/" (truncated))
│    │  │  │  │  │   ├───1.61 MB (00.15%) ++ string(length=13334, copies=103, "data:font/opentype;base64,d09GRgABAAAAACb8AA0AAAAAQNwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABHUE9TAAAaFAAADAUAABuAPd0S8kdTVUIAACYcAAAA3QAAAX41JDWOT1MvMgAAGRgAAABUAAAAYHkZP2tjbWFwAAAZbAAAAGwAAACSQ/FjK2dhc3AAABoMAAAACAAAAAj//wAEZ2x5ZgAAATAAABUhAAAefBjHfBZoZWFkAAAXYAAAADYAAAA2/JJmh2hoZWEAABj4AAAAHwAAACQHEALraG10eAAAF5gAAAFfAAAB3NykEZpsb2NhAAAWcAAAAPAAAADwvX/F621heHAAABZUAAAAHAAAACAAhgCYbmFtZQAAGdgAAAAfAAAAIAAjCJtwb3N0AAAZ+AAAABMAAAAg/7gAMnjalVgFYBrJu9+ZTVhiJGRZNgYENrAQD4QQI41ribVxrUbbHm1Ty/XqdlK5nrtVzt3d3fUv567v/3ouWd43u6Slfd5mhZ2Z7/vNZ/OboTDFUhQ6hGuoZMpEUS6PwAgeweOSLxcjX4wgXzT8ejytIc2XVhrnZSu0FfBGfpVpy9iquOq0ZraBPXToUGtv62H4B49DqKW3l6Ipa+Am9DK+kUqlbFQmRc1Bet4tZiN3foHH7eL0PJONRI43Ik6nYjjBbbPm20SugNZBN5V1QaQP97X3zys9jx1QpaUyPfOHOkoPIYvJUnG/wVR6wNSqPorMvvqm3ojaGvWm1DRL+CFsaqhtG4xoa2WuSTRq9seigSSL5pxoigqnTIHP8W14ElBFUzrKCIgoD+IRbQE0LienE53ufMHC6dDsB9Xsl7PQolrpyszdtdV79mRkZKHr99RV76mtzszMUuMeYeYIyj3UMXDr0arxpUvHpSvg/Za27uqxpUvHKERZ0R3UNfgIaKVY0cWU8ht5dMfPP0OLKzCMGvCLFAEHdgCT8BowhB411I6NjI7VzC8vX5nd8fkvX3RkTx6ZWn0EnBX4E8Zol" (truncated))
│    │  │  │  │  │   ├───1.52 MB (00.14%) ++ string(length=37222, copies=39, "data:font/opentype;base64,d09GRk9UVE8AAGz4AAsAAAAAvDQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABDRkYgAAAKYAAAPZIAAEnef8PbpkdQT1MAAEf0AAAjowAAXdbAT/w0R1NVQgAAa5gAAAFfAAAC+gJm/BtPUy8yAAAFAAAAAE4AAABgWkx2zGNtYXAAAAecAAACrQAABACv2bxEaGVhZAAAARAAAAA0AAAANgaYO09oaGVhAAAE4AAAACAAAAAkBtcFFmhtdHgAAAFEAAADmQAABwCiRinabWF4cAAAAQgAAAAGAAAABgHAUABuYW1lAAAFUAAAAksAAATdPUk0zHBvc3QAAApMAAAAEwAAACD/mgA8AABQAAHAAAB4AWNgZGBgYG9ke+Tg9Dqe3+YrAzPzCwYguGRsHQ+j/2n/Z2N5ybwUyGVmYAKJAgBsMwxyeAGNlAOwZGcQhU/3XXt35tn2W9u7sW3btm3bhdhOKbZt27Zvvp6Hyj5UMlVfnR/3V5/u8TYtVPzuB7AJWmyHq9ana7QPUEWylQr8fFXqBdXbhmqyjWT2lVp8Au1yxlbQZCvVVLs//dq3UR1rh7Eua58q4yurwF5Wkbfz3Tv0x6iS88pZNy239lgV29tqQGvtYo1JWtXuV3HuBlrTd+CM+9FxWtP+Rh+m/xntpbS+rZi+4vsw5lo72UNN/rHWSAYyv6pWz+lOzHF3L9Iof0Kr+vkamZyhsb6dMr6OhvtstdvaWmj3y9BiX06tdnOa2o861H5WlTdpvj2ucm9jj8Eqt51ZW6SstzJeqNmWSZ/2gbQzWjaZroXeBvV8Px5tVpUdw96X8bYjlWFurv1BTD7RcEs1hnUJezfYOcyXa+1QX171itg3a6ncuaZ5vgssqwH6TnPsOb7/kbOWUpvtjx8lxHE3NUbsY4y5nWwD1s3Dk3U1xhYpy9jp3qq5fg5xdsa/1pmcN8IuUDHrS4nlCP9bI5JRfHuhKiPufZFsmH4WXoQP/8aWSj8KL" (truncated))
│    │  │  │  │  │   ├───1.52 MB (00.14%) ++ string(length=37862, copies=39, "data:font/opentype;base64,d09GRk9UVE8AAG7YAAsAAAAAvtwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABDRkYgAAAKYAAAPuwAAEwkBxCxzUdQT1MAAElMAAAj0AAAXaDULsEnR1NVQgAAbRwAAAG6AAADcDLGLR9PUy8yAAAFMAAAAE0AAABgWkh2pGNtYXAAAAegAAACrAAABAjCzatUaGVhZAAAARAAAAA0AAAANgaDQtZoaGVhAAAFEAAAACAAAAAkBsIE/GhtdHgAAAFEAAADywAAB3ioHUI7bWF4cAAAAQgAAAAGAAAABgHeUABuYW1lAAAFgAAAAh4AAASG3UOrqXBvc3QAAApMAAAAEwAAACD/mgA8AABQAAHeAAB4AWNgZGBgYG9ku7l/WlI8v81XBmbmFwxAcMnYXglG/wv5z8aymnkZkMvMwAQSBQBtvQwteAGNlQOwHGkUhc+9PbGfZ+bZtq3Ytm2nvIW1bdu2nbVtK7aT3nPrTS/iqfrq/P7/qx4NwwjY76c2JBzDpC9ytByRsh25TjUSdTnS8SzypQwVxCdrUaD1yMbPiGW/HN+iSC5x9+oYFMooRMgWpMiHCGgP+OVZZGo7jr/A8T+RzXYa7ymB7Z2PoLyDFGqOXI48Pci1KxGptZimc1Gmd2Ka7Cbr2H+V/c/YDmCOBN31WsX2V5jmjEK5voEpuofz8VxnWs05vl22IkIfwQSdhzBnIc8tg19zEKZFtKWYdl6CDlS/9kOdnOu6sgOXy5fI1Q4YKk/xLV2QKetIHSJkF/s9MBT70Beb3R/lU7a3YpTjxwgNIz0534vamevHY4RchSyZBD/HB8n76KnPoBc1Sj5DF7kT2TKH9sdiGdWvg1AR8v1wu5dnl9OWYu0IxcOoYr+Vb5vrTEctfZUs3yKPezLN9zbGuRUyAdkWExmKRNoUY7ZodwzS09Bk8WH/JgHC5GykcH+Onkk/PEye5tqLUGx+PxpOgbvVYmFx+B8B9y+LBXUP2acXI+DF4" (truncated))
│    │  │  │  │  │   ├───1.37 MB (00.13%) ++ string(length=34386, copies=39, "data:font/opentype;base64,d09GRk9UVE8AAGSoAAsAAAAAlYgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABDRkYgAAALcAAAStYAAFgu7TO6FEdQT1MAAFZIAAAMkgAAJDbInOaXR1NVQgAAYtwAAAHJAAAFmiRqBERPUy8yAAAE0AAAAE0AAABgVF91b2NtYXAAAAb0AAAEZwAABvgjj2q+aGVhZAAAARAAAAAxAAAANgb0vVBoaGVhAAAEsAAAACAAAAAkByAE62htdHgAAAFEAAADagAABwi8YkVSbWF4cAAAAQgAAAAGAAAABgHCUABuYW1lAAAFIAAAAdIAAAPkc8FdGXBvc3QAAAtcAAAAEwAAACD/3QAZAABQAAHCAAB4AWNgZGAAYRPDH07x/DZfGZiZXzAAwSUTm0sw+t+Z/2wsK5mPAbnMDEwgUQBQqQyhAAAAeAFtlQOwpEkQhLOq1959HD/bONu2bdu2bdu2bQfOtm3fzX+ZcfMmehERX1QbVdnV9gsmWS1gq2EXWwE9/hUW9S7MF9ZAwR/BMJ5FwS7FdqTAsXOFRXGA1aDVFkYv7Qr2YfKJL4Bhuwxz2xfosG9p/0aFPYdm78HC9hrXzGIu78f8nLMJWZRrHEoOkLXPMHe4CSv4wckfvgHqfR90+p20i5AUUn4X6x+h3pZAoz2ASb4528chHTZD3p9F2t9hfw/HvY1WP4p9XVjA18Q0zkv5dsmX4RLA10l+9+WSn305dNuVON4+xGTabu6/F+8EPMu1/0SHpzHOHuK550KzfU0W4x4T0eaDqKCPGmxCcpuPZjmNTFieffOQDMe3oNpncPxByNsp9MFK3HtepO275O8QkqL9Th9/nyT2Cvf/DItyz7dpl5AP5HeeeRHSjHuTbWineVdyM8cszHITrkePnc7z3YdG/Croz7m47+LIcQzIdKFY2Cf0x0o4SzHjnIzgvJ7wBbYe9QwK3HMBjt2GvEJ2IJuSzUvlJ8l25AhyFTmS3KU+zkfYNHnGd" (truncated))
│    │  │  │  │  │   ├───1.37 MB (00.13%) ++ string(length=35610, copies=39, "data:font/opentype;base64,d09GRk9UVE8AAGhAAAsAAAAAmkAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABDRkYgAAALgAAATeoAAFn5W3/jskdQT1MAAFlsAAANIwAAJ1LA1ubuR1NVQgAAZpAAAAGuAAAFcqHCm5ZPUy8yAAAE3AAAAFEAAABgVGN1YWNtYXAAAAcIAAAEXgAABuZjAEInaGVhZAAAARAAAAAyAAAANgazmo1oaGVhAAAEuAAAACQAAAAkB7QIdGhtdHgAAAFEAAADdAAABtSKmiibbWF4cAAAAQgAAAAGAAAABgG1UABuYW1lAAAFMAAAAdYAAAQdt5hvnHBvc3QAAAtoAAAAFQAAACD/0QAZAABQAAG1AAB4AWNgZGAA4RM/+TXi+W2+MjAzv2AAgkvG2mUw+t+8/2ws05h3MzAxMAMxEAAASakL1AAAeAFtlQOMbkkQhU/VfTbG82zbtv0mWFvB2maw9gZr27Zt2zt4Y3tqT3XyJz1I8qWq3VV9bl2pQT/pC8geLJdtWK/7sFqnIT3Jwxx9E2n4BHPlHuyUe+wjzj0iWY0zJMM+kJXIlAzkyl92nY7AaHkIafKt1cuPVirl9p98YSWahonyG7prN/TV/hgT5nMt97iInOFW8nFw8ji26cH2lO6wfN0fE/Q62lGWL21WoLew/SH9RVYiL9v3ehr9GstPNnPsMftT38EknxvsQbTZ6K3zefebGMcOjEwuxlxdbI/pHIzUaVYg92G2/IWxcp/9wPNPZ0zAJ4x7MNKlDtlyNYbST5MHMFTG2XHyh/2hYxnrCIyS/nahCrKEsSQzuN9UjOSaNO1pb0kL11zH9slWLtsxTWcw3vetQT8C5Df7Sn6xa+RFq2K883jmL5JvpTz/XOY9h/mewrsV4yWbTT+N/hOc043+YDyC9XITeRn9UetYk66143Q571ZjhZw3g/Y/5na3/Mu9duBmfGJvyE32NtctYs6akkLM6fYh5vDMOs7/V2pkLu135EvyK" (truncated))
│    │  │  │  │  │   ├───1.05 MB (00.10%) ── string(length=11, copies=45684, "127.0.10.80")/gc-heap/latin1
│    │  │  │  │  │   ├───1.05 MB (00.10%) ── string(length=5, copies=45679, "socks")/gc-heap/latin1
│    │  │  │  │  │   ├───1.01 MB (00.10%) ++ string(length=1056134, copies=1, "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA2AAAAG8CAYAAAC48cbZAAAABHNCSVQICAgIfAhkiAAAIABJREFUeF7MvXnQpddZ4Pfc/X7323pf1Fq6JVmbZXlfJNnYxjYMW7xCKAZDUZNZKmFqUkwqqUwFipqaqhQ1k/wxmaJSk7AkATIMdioMDIYxNhgb7zLGNpa1S61u9b58+91vfr/n3Lf1qSU7hoAz5+vb977ve9bnPPt5znlrH/2Vw7Mj/QuxenoxGme34/zDk2icjFhej6gNImatiPZyLVoLzRiPR3mvzb1ZoxaDGnn421mfxnQY0WpyPY2I7Yjm2Gfc60RM6hGjnYhunTutWbR7tdjcnvE0orFI+ZjFbCuiTtkp5awiyzabMYpJXJrN4oZX0+7bIxaON6NzJ5Xu2YnpaBpWOZk2YjKZRaNBvyxo+VqNa25Q22xK/fxNqbhOX+o1/vMJNybey+tZ1KisTrnReJLPLZ7PqHM8ZozU02DcZMn+Rf72F5XUGI+/HZYfx8JnRt+jTjm6nPdH5Onvj/h6M/q/ez5OfmQaLWFnfaR59/N7XnMWq1Lp+a4b85/zJvNqd/4X5/z279iWEPSbaYmxP26KOPhgPXp895vTGAKq0Qh8Wakn/MbMSQ24NFt82gKQQgln4D+ZZJ5GvRFccs19Po1GPZpNWyrzNhwy68xnwnkOkMl4Vua4yRwBmToV+KjBNeArdVD3ZDSJ0dC54iZArYmT9NGu572ss1Sa0zUHlvNqP8SJMW2ZvGdyCPZ5TN3igdetlnlLn9rtetYzGvIf/8ShUkcNfAZGwMfUsS8Ms0WfzTiSRubj8/mAfBPuiXetNs/sNM+btCWej6hLfM0yfBxH4hf/anTKfgm37Az/OsDfPEMmyf743LKDPnMJTKRj6bMu3tMnYTnq0wbjsOnsH31pNusx3mGehKMwhb5nEP/E70tQFm0tHGhEvcPDbRgE+A6xZyelu/FgFi0LUtegP6SOOvBrxmSDm" (truncated))
│    │  │  │  │  │   ├───1.00 MB (00.09%) ── string(length=3, copies=43607, "GET")/gc-heap/latin1
│    │  │  │  │  │   ├───0.91 MB (00.09%) ++ string(length=6482, copies=116, "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAN1wAADdcBQiibeAAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAABJpSURBVHic7Zt7jCRHfcc/Vd3z3N3Z1+3tvXw+G4PP+AH4AX6cwTjGF2EZpCQm5KFgoihESXACtsH/EIMVgYFgQqREIlHiSCZIWEkwsogNAfzkfFwMts+YM3c+39t3e3v7mNl5dXdV/fJH98z0zM7eLtzlrCipUamrqx/1+37r9/vVr6tqlIjwfznp11uA1zv9PwGvtwCvd/Jfz8Yv/4jK5Ib9d2vhJlGcrRWrBcZFJA+qgmJWwQmxvOBEPfzMfdHzcpqdlno9nOC1d6mNiP8VRN+4eeNb3VUXbx1YPbJBjRTGGciPgIJaUKZcP8GJymvsPvIT8+LuZ5uzlWmDkn9plu3Hnv2qRKdDljNOwDV35i7ytHrst2/86NiV5/+qLhXG0cpHK42nPLT2QBSCxTpLZAPqUYVqMM/hmd18578ebO7a+9wLTc9sffZeKZ+qPGeUgC0fz15eGBj4/p//xr1DG1adpyIbIIATA6Ji8IBCASDicOLQOnZVtaDMQjDHthcfjZ56/tEjpYbZ/B9/I8GpyHRGfYB47rYP3vAnQ4MDw+rQ/Ms4cXjKx9Nx1srrgEdwYrHOYGyIEQMIkW1y8ZuuyBw89srk/sN7bgG+dioynVECNNywYeJcdbSyB+sMns4kwH087aHQKAUi0ibAicXYCCsRxoRELiQ0ddat21A4ePSVD/C/hYAtn1TrxobWj2tPMd84ngLvoZVGoVBKA0JsloITh3Gmowkuoh6WUUoxOboBFOtOVa4zRoA4/+zR0nhkXJBdCOfwdQZf+SidgCc+Cg4hsX8czhmsszgxNG0dY0NK+VUYYxConKpcZ4yAQczOQ1P78mPF9WS9HIGpEyKIO" (truncated))
│    │  │  │  │  │   ├───0.75 MB (00.07%) ── string(length=7, copies=32736, "0.0.0.0")/gc-heap/latin1
│    │  │  │  │  │   ├───0.67 MB (00.06%) ── string(length=15, copies=29134, "HTTP/2.0 200 OK")/gc-heap/latin1
│    │  │  │  │  │   ├───0.48 MB (00.05%) ── string(length=5, copies=21106, "image")/gc-heap/latin1
(...)
│    │  │  │  │  ├───19.01 MB (01.80%) -- string(<non-notable strings>)
│    │  │  │  │  │   ├──14.07 MB (01.33%) -- malloc-heap
│    │  │  │  │  │   │  ├──14.07 MB (01.33%) ── latin1
│    │  │  │  │  │   │  └───0.00 MB (00.00%) ── two-byte
│    │  │  │  │  │   └───4.94 MB (00.47%) ++ gc-heap
│    │  │  │  │  ├───17.93 MB (01.69%) -- string(length=119470, copies=153, "data:font/woff;base64,d09GRgABAAAAAV3wABIAAAACw4QAAgR+AAAAAAAAAAAAAAAAAAAAAAAAAABGRlRNAAABlAAAABwAAAAcYgkChUdERUYAAAGwAAAAZgAAAHopcybcR1BPUwAAAhgAADFTAACDLI1iZg5HU1VCAAAzbAAACqMAABSexz3ey09TLzIAAD4QAAAAVgAAAGCh07GaY21hcAAAPmgAAANIAAAErjOWwQZjdnQgAABBsAAAAEIAAABCBE4k2GZwZ20AAEH0AAABOwAAAbxT71erZ2FzcAAAQzAAAAAMAAAADAAIABNnbHlmAABDPAAA7ZYAAc58Ibs0ImhlYWQAATDUAAAANAAAADYGmlMfaGhlYQABMQgAAAAhAAAAJAsmCq5obXR4AAExLAAACmwAABOIVOwKJmxvY2EAATuYAAAJnwAACcZJJ9CWbWF4cAABRTgAAAAgAAAAIAcPAhhuYW1lAAFFWAAAAlQAAASGQuBcnnBvc3QAAUesAAAVtwAAMWPsiHvtcHJlcAABXWQAAACKAAAAot02fhoAAAABAAAAAMw9os8AAAAAxPARLgAAAADQ206HeJwdjUEKwgAQAyebPYuPqfYzrejF3j1YsQVFwfpNf+JqQmDIIUHAqnLip2CNuFamspnVIfXqsXbaFx90LB6iQbGJLY7WZ+TRI/bFE+HZt2rufhQ/84VyyYXId35qOf8/fAERxg6SAAB4nO2dCXxURbb/T9UN2UNCCAHXcYMAbig6KoKA8wSVWf7gyN8FRMflufCUUQfHcVdUBpXnMoNxFMImMrLMwICACrTIjiExQAdiCHSn090kne5OOtE3T0z9v/d2JySC/pn5v/+893kvqc/vVt17695bdX7nnDp1u6sjSkTSZZxMlqR/GvHjn8uJtz/64ETp+88P3nmf/HDibQ/fL/8kSdQRY0STqb9hT98+6aFJ0vW+Ox+8X/KcI+JsOSPJ0tXZV5Lv1E5Sey+794pf3/0CR9Kli5wlV1Oqk" (truncated))
│    │  │  │  │  │   ├──17.93 MB (01.69%) ── malloc-heap/latin1
│    │  │  │  │  │   └───0.00 MB (00.00%) ── gc-heap/latin1
│    │  │  │  │  ├───17.93 MB (01.69%) -- string(length=119666, copies=153, "data:font/woff;base64,d09GRgABAAAAAV6EABIAAAACvqgAAgRNAAAAAAAAAAAAAAAAAAAAAAAAAABGRlRNAAABlAAAABwAAAAcYgkCmEdERUYAAAGwAAAAZgAAAHopdSbeR1BPUwAAAhgAAC6BAAB0rl9Hb9hHU1VCAAAwnAAACqUAABSexqfgUE9TLzIAADtEAAAAVwAAAGCgp7GiY21hcAAAO5wAAANIAAAErjOWwQZjdnQgAAA+5AAAAEwAAABMJEEG5WZwZ20AAD8wAAABOwAAAbxn9FyrZ2FzcAAAQGwAAAAMAAAADAAIABNnbHlmAABAeAAA8KkAAdfo0ESDPmhlYWQAATEkAAAAMwAAADYGLVnvaGhlYQABMVgAAAAhAAAAJAq6CoJobXR4AAExfAAACnEAABOI6ISIlGxvY2EAATvwAAAJpgAACcaWnhz8bWF4cAABRZgAAAAgAAAAIAcPAqNuYW1lAAFFuAAAAlUAAAR6doFSlXBvc3QAAUgQAAAVtAAAMWKKISvdcHJlcAABXcQAAAC9AAAA23Sgj+wAAAABAAAAAMw9os8AAAAAxPARLgAAAADQ206aeJwdjTEKwgAUQ19+/iweptrLtKKL3R2s2IKig/WY3sSvCYFHhgQBq8qJn4I14lqZymZWh9Srx9ppX3zQsXiIBsUmtjhan5FHj9gXT4Rn36q5+1H8zBfKJRci3/mp5fz/8AUSDg6WAAB4nO2dCXwUVb7v//9TCQmhQ8KSEBlEVMIiuDI6qKjgXAXBGS+ovBFlxFHfHZfBGZ1xX6KiICKDMipu7CKjwGQUBFQQkbAEAglLCLEJEJruhnTSnU3HK+a8b1UWwqJX57137/18bnI+vzqnq07VOf/l/JfqqrSoiCTJTXK/xP3LlVdfJ51vfejeu6XXv917+11ywd23/HGs/IvE0UesFUOlP+KTufX39/1e2t51+71jpaO3R7wtR6SVtPU+q6R7veO06Ly4/gNHJbInSeKluwyhVSKJkik9p" (truncated))
│    │  │  │  │  │   ├──17.93 MB (01.69%) ── malloc-heap/latin1
│    │  │  │  │  │   └───0.00 MB (00.00%) ── gc-heap/latin1
│    │  │  │  │  ├───15.59 MB (01.47%) -- string(length=119774, copies=133, "data:font/woff;base64,d09GRgABAAAAAV7UABIAAAACvhAAAgSAAAAAAAAAAAAAAAAAAAAAAAAAAABGRlRNAAABlAAAABwAAAAcYgkCkkdERUYAAAGwAAABNAAAAcRmFmBcR1BPUwAAAuQAAC6KAABeTnsohtlHU1VCAAAxcAAADPsAADDo6mgGW09TLzIAAD5sAAAAVwAAAGChC7GwY21hcAAAPsQAAANIAAAErjOWwQZjdnQgAABCDAAAAFYAAABWBJcrSmZwZ20AAEJkAAABOwAAAbx7+WGrZ2FzcAAAQ6AAAAAMAAAADAAIABNnbHlmAABDrAAA7cYAAc+Qp89n62hlYWQAATF0AAAAMwAAADYGYlmraGhlYQABMagAAAAhAAAAJArvCptobXR4AAExzAAACl4AABOIJPNFAGxvY2EAATwsAAAJoAAACcZg7ufmbWF4cAABRcwAAAAgAAAAIAcPAp9uYW1lAAFF7AAAAoEAAAUQVmT86nBvc3QAAUhwAAAVtwAAMWPsiHvtcHJlcAABXigAAACrAAAAzBux+DYAAAABAAAAAMw9os8AAAAAxPARLgAAAADQ206UeJwd0UlL1WEUx/HfOc8/8bZoaeCQWzV9BZZbF7qz16KBgnKvGThlGA44mwPOiqJJ16GNcwn5CoRScNg6o371fHng8+wOHJmkV5Il9TSuYpnivIRq+X0iUx25PqseN6gRN6kZf1EL/qpW/E1tuJ1MHeTqVBfuVg/uVR/uJ9eABvF3DeFhMo2Qa1TjeEKTeErTeEazeE7zeEGLeIlMP8i1rJ84qRW8qjW8Tq5f2sLb2sG7+of/k+tIJ/hUZ/hcV/haN/iWTHd6kJvshcxSLIZfWhp+bek4wzJxlr3B2ZaH31o+LrBC/M7e4yIrwaVWhj/4hsw3fU/uv/0P3ve/+MCP8LGzg5/7Bb70e/wQUmUhFnLkITdU4I+hEleFalwT4jgRlcuiimhMHo1Hh+wcPV9Mj+3tVOV4nO2ceXxU1f33v" (truncated))
│    │  │  │  │  │   ├──15.59 MB (01.47%) ── malloc-heap/latin1
│    │  │  │  │  │   └───0.00 MB (00.00%) ── gc-heap/latin1
│    │  │  │  │  ├───12.50 MB (01.18%) -- string(length=62526, copies=200, "data:application/font-woff;base64,d09GRgABAAAAALcTAA4AAAABY9AAAAAAAAC1qAAAAWsAAALYAAAAAAAAAABPUy8yAAABnAAAAFgAAABgjga5umNtYXAAAAH0AAAG6AAAEgCbcKQqY3Z0IAAAs/gAAAAoAAAAKAjhCLJmcGdtAAC0IAAAANAAAAEs84Klmmdhc3AAALPoAAAAEAAAABAAGgAjZ2x5ZgAALmQAAIWEAAEDFE+Ll4hoZWFkAAABZAAAADYAAAA2CfaG5WhoZWEAAAjcAAAAIQAAACQH6Ae4aG10eAAACQAAAAaSAAAR9o9pil9sb2NhAAAlxAAACJ8AAAj+ol1dpm1heHAAAAFEAAAAIAAAACAE+ATJbmFtZQAAD5QAAAG0AAAEQXAVNDJwb3N0AAARSAAAFHoAACt3rYf6aXByZXAAALTwAAAAtwAAANzHz3zdAAEAAAR+AGAABwBgAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxLy8SDRfDzz1AAMD6AAAAADTwC3PAAAAANQAFJr+i/6dBJIENwAAAAcAAgAAAAAAAHjaY2Bm0mScwMDKwMLUxRTBwMDgDaEZ4xhEGBUZGJgYWNmYQRSLAgNTO1CekQEKcnOKixkcGBh+szDP+K/HaMA8g+GSAgPjfJAc42Om2UBKgYEZAAzUDP142u1WB3NVxxk9Z79PgApVEoh2ue9hid4NohtME92UYGeSIbbHYEyxAePE2LQ4EAc7dLDpFr0pMcVgEBJF9N5Ff0h6mGJjOo/OyyIrECbJ5AdEe+fb/XZ3du6dc/c75wAwAMRGlO0JhIZB8wUA5rGTvMhjd/KjOpqgFd5CT/TFMA7nOE7kLM7hcl7kTwyakqaWiTe/NgPNcpNidphdZo85KyoRUl6GyZ/lC/mrjJfJMkeWyHJJkf1KjdCCGq3VtbZ21E76W/2dTtIZukovO+KEO2Wcss7rzltOH2eUM9u574a50a7jet04t7pby63vNnabu" (truncated))
│    │  │  │  │  │   ├──12.50 MB (01.18%) ── malloc-heap/latin1
│    │  │  │  │  │   └───0.00 MB (00.00%) ── gc-heap/latin1
│    │  │  │  │  ├───12.50 MB (01.18%) -- string(length=63162, copies=200, "data:application/font-woff;base64,d09GRgABAAAAALjxAA4AAAABY7AAAAAAAAC3iAAAAWkAAALUAAAAAAAAAABPUy8yAAABnAAAAFgAAABgjhO5w2NtYXAAAAH0AAAG6AAAEgCbcKQqY3Z0IAAAtdgAAAAoAAAAKAkBCMBmcGdtAAC2AAAAANAAAAEs84Klmmdhc3AAALXIAAAAEAAAABAAGgAjZ2x5ZgAALmAAAIdnAAEDGCHBWq5oZWFkAAABZAAAADYAAAA2Cc5QaWhoZWEAAAjcAAAAIQAAACQIAwfMaG10eAAACQAAAAaXAAAR9rQtdihsb2NhAAAlwAAACJ0AAAj+mMNT+G1heHAAAAFEAAAAIAAAACAE+ATJbmFtZQAAD5gAAAGrAAAEHYmAkptwb3N0AAARRAAAFHoAACt3rYf6aXByZXAAALbQAAAAtwAAANzHz3zdAAEAAAR+AGAABwBgAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxK7mro5fDzz1AAMD6AAAAADTwC3PAAAAANPJ3iv+f/6QBKwENwAAAAcAAgAAAAAAAHjaY2BmMmKcwMDKwMLUxRTBwMDgDaEZ4xhEGBUZGJgYWNmYQRSLAgNTO1CekQEKcnOKixkcGBh+szDP+K/HaMs8g+GSAgPjfJAc42Om2UBKgYEZABFEDRN42u1WB3NVxxk9Z79PgApVEoh2ue9hid4NohtME92UYGeSIbbHYEyxAePE2LQ4EAc7dLDpFr0pMcVgEBJF9N5Ff0h6mGJjOo/OyyIrECbJ5AdEe+fb/XZ3du6dc/c75wAwAMRGlO0JhIZB8wUA5rGTvMhjd/KjOpqgFd5CT/TFMA7nOE7kLM7hcl7kTwyakqaWiTe/NgPNcpNidphdZo85KyoRUl6GyZ/lC/mrjJfJMkeWyHJJkf1KjdCCGq3VtbZ21E76W/2dTtIZukovO+KEO2Wcss7rzltOH2eUM9u574a50a7jet04t7pby63vNnabu" (truncated))
│    │  │  │  │  │   ├──12.50 MB (01.18%) ── malloc-heap/latin1
│    │  │  │  │  │   └───0.00 MB (00.00%) ── gc-heap/latin1
│    │  │  │  │  ├───12.50 MB (01.18%) -- string(length=63198, copies=200, "data:application/font-woff;base64,d09GRgABAAAAALkLAA4AAAABZUwAAAAAAAC3oAAAAWsAAALXAAAAAAAAAABPUy8yAAABnAAAAFgAAABgjfO5sGNtYXAAAAH0AAAG6AAAEgCbcKQqY3Z0IAAAtfAAAAAoAAAAKAiyCJlmcGdtAAC2GAAAANAAAAEs84Klmmdhc3AAALXgAAAAEAAAABAAGgAjZ2x5ZgAALlwAAIeDAAEEtIo/tthoZWFkAAABZAAAADYAAAA2Ca9QlmhoZWEAAAjcAAAAIQAAACQHxAeeaG10eAAACQAAAAaWAAAR9mCMpmJsb2NhAAAlvAAACJ4AAAj+fhw42m1heHAAAAFEAAAAIAAAACAE+ATJbmFtZQAAD5gAAAGnAAAEIPrebOFwb3N0AAARQAAAFHoAACt3rYf6aXByZXAAALboAAAAtwAAANzHz3zdAAEAAAR+AGAABwBgAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxNlYD91fDzz1AAMD6AAAAADTwC3PAAAAANPJ3jj+nP6wBHAENwAAAAcAAgAAAAAAAHjaY2BmkmecwMDKwMLUxRTBwMDgDaEZ4xhEGBUZGJgYWNmYQRSLAgNTO1CekQEKcnOKixkcGBh+szDP+K/HKMs8g+GSAgPjfJAc42Om2UBKgYEZAAd9DOB42u1WB3NVxxk9Z79PgApVEoh2ue9hid4NohtME92UYGeSIbbHYEyxAePE2LQ4EAc7dLDpFr0pMcVgEBJF9N5Ff0h6mGJjOo/OyyIrECbJ5AdEe+fb/XZ3du6dc/c75wAwAMRGlO0JhIZB8wUA5rGTvMhjd/KjOpqgFd5CT/TFMA7nOE7kLM7hcl7kTwyakqaWiTe/NgPNcpNidphdZo85KyoRUl6GyZ/lC/mrjJfJMkeWyHJJkf1KjdCCGq3VtbZ21E76W/2dTtIZukovO+KEO2Wcss7rzltOH2eUM9u574a50a7jet04t7pby63vNnabu" (truncated))
│    │  │  │  │  │   ├──12.50 MB (01.18%) ── malloc-heap/latin1
│    │  │  │  │  │   └───0.00 MB (00.00%) ── gc-heap/latin1
│    │  │  │  │  └───12.50 MB (01.18%) -- string(length=63230, copies=200, "data:application/font-woff;base64,d09GRgABAAAAALklAA4AAAABZegAAAAAAAC3vAAAAWkAAALVAAAAAAAAAABPUy8yAAABnAAAAFgAAABgjeW5qGNtYXAAAAH0AAAG6AAAEgCbcKQqY3Z0IAAAtgwAAAAoAAAAKAiQCIVmcGdtAAC2NAAAANAAAAEs84Klmmdhc3AAALX8AAAAEAAAABAAGgAjZ2x5ZgAALnAAAIeKAAEFSNlYQZJoZWFkAAABZAAAADYAAAA2CdGEtGhoZWEAAAjcAAAAIQAAACQHogeNaG10eAAACQAAAAajAAAR9j47uudsb2NhAAAl0AAACJ8AAAj+0pqNQG1heHAAAAFEAAAAIAAAACAE+ATJbmFtZQAAD6QAAAGvAAAEJhmpQCRwb3N0AAARVAAAFHoAACt3rYf6aXByZXAAALcEAAAAtwAAANzHz3zdAAEAAAR+AGAABwBgAAYAAgACAB4AAgAAAGQD6AAEAAEAAQAAAAMAxJmWFwlfDzz1AAMD6AAAAADTwC3PAAAAANQAEkj+of6+BFcENwAAAAcAAgAAAAAAAHjaY2BmEmecwMDKwMLUxRTBwMDgDaEZ4xhEGBUZGJgYWNmYQRSLAgNTO1CekQEKcnOKixkcGBh+szDP+K/HyM88g+GSAgPjfJAc42Om2UBKgYEZAANTDMp42u1WB3NVxxk9Z79PgApVEoh2ue9hid4NohtME92UYGeSIbbHYEyxAePE2LQ4EAc7dLDpFr0pMcVgEBJF9N5Ff0h6mGJjOo/OyyIrECbJ5AdEe+fb/XZ3du6dc/c75wAwAMRGlO0JhIZB8wUA5rGTvMhjd/KjOpqgFd5CT/TFMA7nOE7kLM7hcl7kTwyakqaWiTe/NgPNcpNidphdZo85KyoRUl6GyZ/lC/mrjJfJMkeWyHJJkf1KjdCCGq3VtbZ21E76W/2dTtIZukovO+KEO2Wcss7rzltOH2eUM9u574a50a7jet04t7pby63vNnabu" (truncated))
│    │  │  │  │      ├──12.50 MB (01.18%) ── malloc-heap/latin1
│    │  │  │  │      └───0.00 MB (00.00%) ── gc-heap/latin1
│    │  │  │  └────6.55 MB (00.62%) ++ (13 tiny)
│    │  │  └───25.37 MB (02.40%) ++ active/window(moz-extension://074dcd7d-9b54-45eb-a95b-010f0825867f/background.html)
│    │  └────0.02 MB (00.00%) ++ js-non-window/zones/zone(0x7f86f247d000)/compartment(moz-extension://074dcd7d-9b54-45eb-a95b-010f
```
